pgcli_sublime.py

```diff
@@ -461,10 +466,17 @@ def run_sqls_async(view, sqls):
 
 
 def run_sql_async(view, sql, panel):
-    executor = executors[view.buffer_id()]
+    executor = executors[view.id()]
     logger.debug('Command: PgcliExecute: %r', sql)
     save_mode = get(view, 'pgcli_save_on_run_query_mode')
 
+    try:  # Check if the connection has died
+        executor.conn.cursor().execute('select 1')
```

Comment:
This is the only part I'm unsure of. Can we move the `if executor.conn.closed` check to the main `except psycopg2.DatabaseError` below, and avoid the constant overhead of an additional command on every execution?